Extensions/Physics2Behavior/tests/physics2runtimebehavior.spec.js

```diff
@@ -0,0 +1,128 @@
+// TODO: find a way to add assertions at events time to enable this test.
+describe.skip('Physics2RuntimeBehavior', () => {
```

Comment:
This needs to be addressed.
Arrays clearing can be done at the beginning of the preEvent function.

Reply:
The conditions use the functions `hasCollisionStartedBetween` and `hasCollisionStoppedBetween`, and themselves use the arrays `contactsStartedThisFrame` and `contactsStoppedThisFrame` so I can't erase them in the pre events, or is there something that I'm missing?

Reply:
It can be in this order:
- preEvent
  - clear the arrays
  - step Box2d (this fill the arrays)
- events (conditions use the arrays)
loop...